libs/tx-reconciliation/src/cache.ts

```diff
@@ -101,15 +101,17 @@ const mkCachePath = (path?: string) => {
   return cachePath;
 }
 
-const convertTimestamp = (obj: any) =>
+const convertTimestamp = (obj: any, prefix = "") =>
   obj
-    ? Timestamp.fromMillis(obj._seconds * 1000 + obj._nanoseconds / 100000)
+    ? Timestamp.fromMillis(obj[`${prefix}seconds`] * 1000 + obj[`${prefix}nanoseconds`]  / 100000)
     : undefined;
 const convertTimestamps = (tx: any) => {
   if (tx) {
```

Comment:
woops, not meant to be in this commit.

This is mostly obsolete code, so just ignore it.